docs/dev-tools/chopsticks/index.md

```diff
@@ -0,0 +1,395 @@
+  const api = await ApiPromise.create({ provider: wsProvider });
+  
+  // Now you can use 'api' to interact with your fork
+  console.log(`Connected to chain: ${await api.rpc.system.chain()}`);
+}
+
+connectToFork();
+```
+
+## Replaying Blocks
+
+Chopsticks allows you to replay specific blocks from a chain, which is useful for debugging and analyzing state changes.  You can use the parameters in the [Configuration](#configuration) section to set up the chain configuration, and then use the run-block subcommand with additional options:
+
+|    Option     |           Description           |
+| :-----------: | :-----------------------------: |
+| `output-path` |    File path to print output    |
+|    `html`     | Generate html with storage diff |
+|    `open`     |       Open generated html       |
+
+For example, to replay block 1000 from Polkadot and save the output to a JSON file:
+
+```bash
+npx @acala-network/chopsticks@latest run-block  \
+--endpoint wss://polkadot-rpc.dwellir.com  \
+--output-path ./polkadot-output.json  \
+--block 1000
+```
+
+## XCM Testing
+To test XCM (Cross-Consensus Messaging) messages between networks, you can fork multiple parachains and a relay chain locally using Chopsticks. 
+
+|    Option    |      Description       |
+| :----------: | :--------------------: |
+| `relaychain` | Relaychain config file |
+| `parachain`  | Parachain config file  |
+
+
+For example, to fork Moonbeam, Astar, and Polkadot enabling XCM between them, you can use the following command:
+
+```bash
+npx @acala-network/chopsticks@latest xcm \
+--r polkadot \
+--p moonbeam \
+--p astar
+```
+
+After running the command, you should see output similar to the following:
+
+```bash
+[13:46:07.901] INFO: Loading config file https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/moonbeam.yml
+    app: "chopsticks"
+[13:46:12.631] INFO: Moonbeam RPC listening on port 8000
+    app: "chopsticks"
+[13:46:12.632] INFO: Loading config file https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/astar.yml
+    app: "chopsticks"
+        chopsticks::executor  TRACE: Calling Metadata_metadata
+        chopsticks::executor  TRACE: Completed Metadata_metadata
+[13:46:23.669] INFO: Astar RPC listening on port 8001
+    app: "chopsticks"
+[13:46:25.144] INFO (xcm): Connected parachains [2004,2006]
+    app: "chopsticks"
+[13:46:25.144] INFO: Loading config file https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/polkadot.yml
+    app: "chopsticks"
+        chopsticks::executor  TRACE: Calling Metadata_metadata
+        chopsticks::executor  TRACE: Completed Metadata_metadata
+[13:46:53.320] INFO: Polkadot RPC listening on port 8002
+    app: "chopsticks"
+[13:46:54.038] INFO (xcm): Connected relaychain 'Polkadot' with parachain 'Moonbeam'
+    app: "chopsticks"
+[13:46:55.028] INFO (xcm): Connected relaychain 'Polkadot' with parachain 'Astar'
+    app: "chopsticks"
+```
```

Comment:
Is it possible to replace this bash element with the terminal element?

Just as we did here: https://github.com/papermoonio/polkadot-ecosystem-docs-draft/blob/main/docs/paseo-testnet/onboarding.md?plain=1#L63

Reply:
Yes, I can replace that and see how it looks